src/components/transactions/TxSigners/index.tsx

```diff
@@ -19,18 +22,21 @@ import { isCancellationTxInfo, isExecutable, isMultisigDetailedExecutionInfo } f
 import EthHashInfo from '@/components/common/EthHashInfo'
 
 import css from './styles.module.css'
+import txSignersCss from '@/components/transactions/TxSigners/styles.module.css'
 import useSafeInfo from '@/hooks/useSafeInfo'
+import palette from '@/styles/colors'
```

Comment:
This does not take dark mode into account:

![image](https://user-images.githubusercontent.com/20442784/210748578-d77c432b-bb74-407c-a85d-f20abc36ad7d.png)